### DISCUSSION TITLE
Hide Menu for specific user?

### DISCUSSION THREAD

**Discussion by nicedevil007** [linuxserver/Heimdall#711] [Ideas]
Heya, I can't find a solution to hide the settings menu to a specific User.

Is that possible in any way?

I just want a user that can create the apps and stick it to the mainpage and is not able to remove/add/move anything from there, so the menu isn't needed.

Can anyone help me out?

**Comment by GAS85** [comment]
I found only way to hide it at all users for now. Simply add Custom CSS in Settings:
```
#config-buttons a {
  width: 0;
}
```
To revert you have to visit `../settings` and remove custom CSS.

**Reply by Ramalama2** [reply]
#config-buttons {
  display: none;
}

with wide 0 it is still visible, display none fixes it.

Pretty old discussion, but since i found it in google, i think that might help others :-)

**Reply by AleixMT** [reply]
> I found only way to hide it at all users for now. Simply add Custom CSS in Settings:
> 
> ```
> #config-buttons a {
>   width: 0;
> }
> ```
> 
> To revert you have to visit `../settings` and remove custom CSS.

What do you mean by visiting `../settings` ? Could you please explain how to revert the changes in order to remove the CSS code and be able to edit the buttons again?

**Reply by GAS85** [reply]
If your heimdall Adress e.g. `https://balbla` to open settings you need to use URL like `https://balbla/settings`

**Reply by AleixMT** [reply]
> If your heimdall Adress e.g. `https://balbla` to open settings you need to use URL like `https://balbla/settings`

Uhhhh it worked like a charm! Thank you for your fast response!!

**Reply by Ramalama2** [reply]
> If your heimdall Adress e.g. `https://balbla` to open settings you need to use URL like `https://balbla/settings`

We could actually make with css a floating button to /items and /serttings actually 😂
Shouldn't be that hard, maybe if i have time i make sth and post here 😂

**Comment by DigitEgal** [comment]
by setting "width; 0" i can still see them.
This custom css makes it invisible:

#config-buttons a {
  display: none;
}

**Comment by abdennour** [comment]
I thought only me thought about "CSS" display:none. .  :) :-)

**Comment by FlyingT** [comment]
Use this to hide the buttons, but make them visible when the mouse is over them:

```
#config-buttons {
  opacity: 0 !important;
  transition: opacity 0.3s ease-in-out !important;
}

#config-buttons:hover {
  opacity: 1 !important;
}
```
